test/client/components/calls/pre_call_btn_grp.test.js

```diff
@@ -26,9 +26,9 @@ describe('Component testing for <PreCallButtonGroup />: ', () => {
     describe('Component rendering: ', () => {
```

Comment:
Do you have reducer tests and action tests?

Reply:
I do now :✖️